src/client.rs

```diff
@@ -34,21 +35,57 @@ pub struct Client {
 #[builder(setter(into))]
 #[builder(name = "ClientBuilder")]
 pub struct ClientBuilderInternal {
+    /// Set an optional Github token.
+    /// This allows for more requests before
+    /// getting rate-limited.
     github_token: Option<String>,
+    /// Check links matching this set of regular expressions
     includes: Option<RegexSet>,
-    excludes: Excludes,
+    /// Exclude links matching this set of regular expressions
+    excludes: Option<RegexSet>,
+    /// Exclude all private network addresses
+    exclude_all_private: bool,
+    /// Exclude private IP addresses
+    exclude_private_ips: bool,
+    /// Exclude link-local IPs
+    exclude_link_local_ips: bool,
+    /// Exclude loopback IP addresses (e.g. 127.0.0.1)
+    exclude_loopback_ips: bool,
+    /// Maximum number of redirects before returning error
     max_redirects: usize,
+    /// User agent used for checking links
     user_agent: String,
+    /// Ignore SSL errors
     allow_insecure: bool,
+    /// Valid URI schemes (e.g. https, http)
     scheme: Option<String>,
```

Comment:
I'd change the doc to say `Allowed` rather than `Valid` and add more context of what it affects (link discovery).

Reply:
Done. 👍